src/deepsparse/v2/utils/state.py

```diff
@@ -62,3 +63,21 @@ def update_value(self, attribute: str, value: Union[str, int, list]):
 
     def update_state(self, value: Any):
         self._current_state.update(value)
+
+    def get_state(self, key):
```

Comment:
```suggestion
    def get(self, key):
```

Reply:
isnt get_state more consistent with the definitions of this class?